### PR TITLE
Add some logging to CustomViewState3dCreator

### PR DESCRIPTION
**Pull request by nick4598**
Main reason for adding this, is we saw an extremely long running getCustomViewState3d RPC operation. There were no daemon downloads going on, or any other logging. These log messages would atleast indicate to us if some specific query was taking a long time.

### CODE REVIEW

File: common/changes/@itwin/core-backend/nick-add-logging-to-viewstatecreator_2022-09-21-19-38.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "Add some logging in the customviewstate3dcreator.",
```
Comment (aruniverse) [inline common/changes/@itwin/core-backend/nick-add-logging-to-viewstatecreator_2022-09-21-19-38.json:5]:
```suggestion
      "comment": "Add some logging in CustomViewState3dCreator.",
```

File: core/backend/src/CustomViewState3dCreator.ts [hunk 2]
```diff
@@ -39,9 +42,10 @@ export class CustomViewState3dCreator {
 
   private async _getAllCategories(): Promise<Id64Array> {
     // Only use categories with elements in them
+    Logger.logInfo(loggerCategory, `Starting getAllCategories query.`)
```
Comment (aruniverse) [inline core/backend/src/CustomViewState3dCreator.ts:45]: nit; none of these logs need backticks for the actual message, theyre not templated

**Review by aruniverse — APPROVED** [review]

File: core/backend/src/CustomViewState3dCreator.ts (hunk 2 above)
Comment (nick4598) [reply core/backend/src/CustomViewState3dCreator.ts]: Changed

File: core/backend/src/CustomViewState3dCreator.ts [hunk 3]
```diff
@@ -39,9 +42,10 @@ export class CustomViewState3dCreator {
 
   private async _getAllCategories(): Promise<Id64Array> {
     // Only use categories with elements in them
+    Logger.logInfo(loggerCategory, "Starting getAllCategories query.")
```
Comment (aruniverse) [inline core/backend/src/CustomViewState3dCreator.ts:45]:
```suggestion
    Logger.logInfo(loggerCategory, "Starting getAllCategories query.");
```

File: core/backend/src/CustomViewState3dCreator.ts [hunk 4]
```diff
@@ -39,9 +42,10 @@ export class CustomViewState3dCreator {
...
     const query = `SELECT DISTINCT Category.Id AS id FROM BisCore.GeometricElement3d WHERE Category.Id IN (SELECT ECInstanceId FROM BisCore.SpatialCategory)`;
     const categories: Id64Array = await this._executeQuery(query);
-
+    Logger.logInfo(loggerCategory, "Finished getAllCategories query.")
```
Comment (aruniverse) [inline core/backend/src/CustomViewState3dCreator.ts:48]:
```suggestion
    Logger.logInfo(loggerCategory, "Finished getAllCategories query.");
```

File: core/backend/src/CustomViewState3dCreator.ts [hunk 5]
```diff
@@ -55,6 +59,7 @@ export class CustomViewState3dCreator {
     if (modelIdsList.length === 0)
       return modelExtents;
     const modelIds = new Set(modelIdsList);
+    Logger.logInfo(loggerCategory, "Starting getModelExtents query.")
```
Comment (aruniverse) [inline core/backend/src/CustomViewState3dCreator.ts:62]:
```suggestion
    Logger.logInfo(loggerCategory, "Starting getModelExtents query.");
```

File: core/backend/src/CustomViewState3dCreator.ts [hunk 6]
```diff
@@ -69,6 +74,7 @@ export class CustomViewState3dCreator {
         continue;
       }
     }
+    Logger.logInfo(loggerCategory, "Finished getModelExtents query.")
```
Comment (aruniverse) [inline core/backend/src/CustomViewState3dCreator.ts:77]:
```suggestion
    Logger.logInfo(loggerCategory, "Finished getModelExtents query.");
```

File: core/backend/src/CustomViewState3dCreator.ts [hunk 7]
```diff
@@ -82,12 +88,13 @@ export class CustomViewState3dCreator {
     const select = "SELECT ECInstanceId FROM Bis.GeometricModel3D WHERE IsPrivate = false AND IsTemplate = false";
     const spatialCriterion = "AND (IsNotSpatiallyLocated IS NULL OR IsNotSpatiallyLocated = false)";
     let models = [];
+    Logger.logInfo(loggerCategory, "Starting getAllModels query.")
```
Comment (aruniverse) [inline core/backend/src/CustomViewState3dCreator.ts:91]:
```suggestion
    Logger.logInfo(loggerCategory, "Starting getAllModels query.");
```

File: core/backend/src/CustomViewState3dCreator.ts [hunk 8]
```diff
@@ -82,12 +88,13 @@ export class CustomViewState3dCreator {
...
     try {
       models = await this._executeQuery(`${select} ${spatialCriterion}`);
     } catch {
       models = await this._executeQuery(select);
     }
-
+    Logger.logInfo(loggerCategory, "Finished getAllModels query.")
```
Comment (aruniverse) [inline core/backend/src/CustomViewState3dCreator.ts:97]:
```suggestion
    Logger.logInfo(loggerCategory, "Finished getAllModels query.");
```

**Comment by nick4598** [comment]
@mergifyio backport release/3.3.x

**Pull request merged**